modules/persistent-storage-snapshots-starting-controller-provisioner.adoc

```diff
@@ -46,6 +46,12 @@ rules:
     resources: ["volumesnapshotdatas"]
     verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
 ----
+. As root, provide the `hostNetwork security context` constraint (SCC):
```

Comment:
`root` ? Perhaps you mean cluster-admin ?

Reply:
Yes, that is more accurate. Updated.